### ISSUE TITLE
DirecTv Remote 

### ISSUE THREAD

**Issue by Shendorock** [GerryDazoo/Slinger#201]
Gerry -

First, huge thanks for doing this.  I'm incredibly happy being able to continue using my Slingbox.  

I'm noticing I'm having a couple of small issues with the DirecTv Remote provided; for some reason the Exit and Last buttons don't seem to function as they should.

Also, might it be possible to add the DirecTv's skip forward and backward buttons?

Many thanks again.

**Comment by GerryDazoo** [comment]
Maybe the author of that remote might help. But you can do it yourself. Read the release notes. Also check this discussion. Makes it easier to find the proper keycodes than just trail and error.
https://github.com/GerryDazoo/Slinger/discussions/187

**Comment by Shendorock** [comment]
Thanks much!

**Comment by GerryDazoo** [comment]
Just another caveat. It is quite possible that he last and exit buttons do work but you don't have the same dtv box as the author.

**Comment by tmcafee52** [comment]
I just retested the DirecTV remote file on my H24 receiver and the Prev (Last) and Exit buttons work properly.  Is it possible that you're not using the correct file because there is no "Last" button in the DirecTV file.  It was replaced with "Prev".  

I also have a modified file with larger rectangular digit buttons and minor color changes.

[remote_directv-h24_large_digits.txt](https://github.com/GerryDazoo/Slinger/files/9905397/remote_directv-h24_large_digits.txt)
